File: 프로그래머스/lv1/12940. 최대공약수와 최소공배수/최대공약수와 최소공배수.js
function solution(n, m) {
    var answer = [];
    if(n>m){
        var min = m
        var max = n
    } else{
        var min = n
        var max = m
    }
    //최대공약수
    for(i=min; i>=0; i--){
        if(min%i == 0 && max%i == 0){
            answer[0] = i
            break
        }
    }
    //최소공배수
    for(i=1; true; i++){
        if(i%n == 0 && i%m == 0){
            answer[1] = i
            break
        }
    }
    return answer;
}

File: 백준/Bronze/2609. 최대공약수와 최소공배수/최대공약수와 최소공배수.py
num1,num2 = map(int,input().split())
max = 0

for i in range(1,num1+1):
  if num1%i == 0 and num2%i == 0:
    if i > max:
      max = i

print(max)

answer = max * num1//max * num2//max

print(answer)